onboarding/source/controller.c

```diff
@@ -94,7 +100,9 @@ static void ledTimerCallback(TimerHandle_t xTimer) {
 static void lightTimerCallback(TimerHandle_t xTimer) {
     /* USER CODE BEGIN */
     // Send light event to light service queue
-
+    ASSERT(xTimer != NULL);
+    
+    sendToLightServiceQueue(MEASURE_LIGHT);
```

Comment:
Print an error message if the `sendToLightServiceQueue` failed

Reply:
Done